addons/mail/static/src/models/thread_viewer/thread_viewer.js

```diff
@@ -276,8 +276,8 @@ function factory(dependencies) {
                 'threadCaches',
             ],
         }),
-        threadCacheInitialPosition: attr({
-            compute: '_computeThreadCacheInitialPosition',
+        threadCacheInitialScrollPosition: attr({
+            compute: '_computeThreadCacheInitialScrollPosition',
```

Comment:
Storing a scroll position in pixel feels very unreliable (subject to change depending on space given to the threadViewer, ...)

Maybe we would want to change this logic to storing something more reliable, such as the "currently seen message". Though it has its share of problems too (target message being deleted, ...).

Reply:
Could be an improvement for later no ? because now it just don't work at all and it brings traceback sometimes

Reply:
Yes definitely not now, maybe never, just brainstorming.

Reply:
https://www.odoo.com/web#id=2287241&action=3206&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720